exercises/concept/wizards-and-warriors-2/src/main/java/GameMaster.java

```diff
@@ -0,0 +1,13 @@
+public class GameMaster {
+
+    // TODO: define the 'describe(Character character)' static method
+
+    // TODO: define the 'describe(Destination destination)' static method
+
+    // TODO: define the 'describe(TravelMethod travelMethod)' static method
+
+    // TODO: define the 'describe(Character character, Destination destination, TravelMethod travelMethod)' static method
+
+    // TODO: define the 'describe(Character character, Destination destination)' static method
```

Comment:
These TODO comments still give away too much IMO, as they contain the full method signature so it's too easy to figure out I think. How about this?

```suggestion
    // TODO: define a 'describe' method that returns a description of a Character

    // TODO: define a 'describe' method that returns a description of a Destination

    // TODO: define a 'describe' method that returns a description of a TravelMethod

    // TODO: define a 'describe' method that returns a description of a Character, Destination and TravelMethod

    // TODO: define a 'describe' method that returns a description of a Character and Destination
```